src/Aurora.Presentation/appsettings.Development.json

```diff
@@ -1,2 +1,8 @@
 {
+  "ConnectionStrings": {
+    "MainDb": "Server=localhost:1434;Database=master;User=SA;Password=password1!;"
```

Comment:
is this repo private ? even if it is, isn't this a security concern ?

Reply:
This is a connection string to the local docker instance, so that you do not have to do nothing with your application to be able to run it with docker environment. If your sa password is password1!, then you really should not be doing sa.